src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,105 @@
-import React from 'react';
 import './NewCommentForm.scss';
+import React, { FormEvent, useState } from 'react';
+import { Comment } from '../../react-app-env';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
+type Props = {
+  commentsList: Comment[] | undefined;
+  setCommentsList: (arg:Comment[]) => void;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+  commentsList, setCommentsList,
+}) => {
+  const [yourname, setYourname] = useState('');
+  const [youremail, setYouremail] = useState('');
+  const [yourcomment, setYourcomment] = useState('');
+  const [error, setError] = useState(false);
+
+  const addComment = (newComment: Comment) => {
+    if (commentsList) {
+      commentsList.unshift(newComment);
+      const copy = [...commentsList];
+
+      setCommentsList(copy);
+    }
+  };
```

Comment:
This function doesn't delete the comment from the database
Move this function to the top level and make it so, this function will delete the comment from the database
also this function should fetch comments after post a comment

```suggestion
  };
```